core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/util/deprecationMessages.kt

```diff
@@ -160,6 +160,8 @@ internal const val TO_CSV = "toCsv() is deprecated in favor of toCsvStr() in dat
 internal const val TO_CSV_IMPORT = "org.jetbrains.kotlinx.dataframe.io.toCsvStr"
 internal const val TO_CSV_REPLACE = "this.toCsvStr()"
 
+internal const val SPLIT_STR = "Shortcut can be trivially replaced"
```

Comment:
This isn't very descriptive. As a user I'd expect reading something like "This shortcut is deprecated in favor of `.by(",").into {}`". That makes it immediately clear what to do about it :)

Reply:
I don't like existing message that much, but i also don't think it's worth to repeat built-in deprecation actions:
![image](https://github.com/user-attachments/assets/b6a1759d-b2ea-45c6-b663-039d034ac203)

![image](https://github.com/user-attachments/assets/6ca88468-c1e4-4484-b421-213d5a70d567)

Reply:
do you have another idea for the message?

Reply:
that looks different than I remember... hmmm

Reply:
so is it ok?

Reply:
"Please explicitly specify how the `String` should be split. This shortcut will be removed in version xxx" (where xxx specifies in which version it will be ERROR)